src/pages/landing/deriv-go/_deriv-go-profits.js

```diff
@@ -0,0 +1,111 @@
+import React from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import { graphql, useStaticQuery } from 'gatsby'
+import device from 'themes/device'
+import { Container, Flex, SectionContainer } from 'components/containers'
+import { Header, Text, QueryImage } from 'components/elements'
+
+const StyledSection = styled(SectionContainer)`
+    background: rgba(243, 246, 250, 1);
+    padding: 8.9rem 0 8rem 0;
+    @media ${device.tabletL} {
+        padding: 40px 16px;
+    }
+`
```

Comment:
We can omit this component and pass it all directly as props. For the media queries, you can pass the style object into prop tablet for this case.